text/0000-guides-restructure.md

```diff
@@ -0,0 +1,397 @@
+
+**Core Concepts**
+
+- What is Ember?
+- Getting Started
+- Anatomy of an app (future new content)
+
+**Fundamentals**
+
+- Template syntax
+- Working with JavaScript
+- Components
+- Routing (includes Routes, Router, and Controllers)
+- State management (aka computed properties)
+- Services
+
+**Leveling up**
+
+- Template helpers
+- Data management (future new content that references separate Ember Data guides)
+- Addons and dependencies
+- Testing
+- Configuration
+- Deploying
+- Upgrading (will contain links to upgrade resources and syntax conversion guides)
+- Debugging (currently named Ember Inspector)
+
+### Topic naming
+
+As much as possible, we aim for the topics to be named after their general web development concepts, and not the Ember-specific implementation of them.
+
+### Logic for ordering and grouping
+
+In this section, we will cover why this order and grouping is being proposed.
+
+#### Groupings
+
+Groupings are added for the benefit of new learners. They break up the content visually and gives them a clue about which sections to pay the most attention to.
+
+Just because something is included in "Leveling up" does not automatically make it an "advanced" topic. The main logic for the division between "Fundamentals" and "Leveling up" are the following tests: Can someone ignore this section and still use Ember effectively? Can it be learned in any order if someone knows the basics? If so, it goes in "Leveling up." 
+
+As we look at each topic, we also ask: Does someone need prior knowledge of another topic in order to understand this section? Whatever those "prior knowledge" topics are, we add them as Fundamentals.
+
+#### Anatomy of an app
+
+The most common points of confusion for new devs have become things like “how does everything fit together? Why can't I call a sibling component's actions directly?” The faster we can get the “Mental Model” of Ember across, the better. The Super Rentals tutorial covers the bases well for using the CLI and adding basic interactions. Anecdotally, thanks to the Tutorial, it seems that the missing pieces in the guides are mostly architectural. "Anatomy of an app" is the main area of new content, where we attempt to give an overview of the topics a web developer has to care about, the parts of Ember that address them, and the file structure. Current Ember developers will reference this section while learning new features.
+
+#### Templates & Template helpers
+
+Templates come before components, since it is possible for someone to not know JavaScript/Handlebars and still contribute to Ember. For example, designers could effectively work in only templates, using plain HTML. To make Templates work as the opening topic, additions are necessary in the introduction.
+
+Template Syntax will cover Handlebars basics and built-in features like `{{input}}`, `{{#each}}`, and `{{#if}}`.
```

Comment:
are we still calling them handlebars long term? 
I remember a discord discussion about this some time back -- due to divergence of all the mental model things that ember's templates fixed when compared to vanilla handlebars. My discord search-fu isn't so great. seems to not be as fuzzy as I need it to be

Reply:
I would suggest to differentiate between control structures (loops and conditionals), block and inline styles and then helpers and components syntax (where each is linking to the dedicated section of the docs).

Second: the `{{input}}` helper is a bad example and should be treated as *classic* or *legacy*, since it is a two-way-binding that'll go away with octane/glimmer arriving, where people are held to use regular `<input>` elements with proper bindings on them. Put them into a deprecated/classic section or better throw them out (but this may be to early 😇)

Reply:
I will remove `{{input}}` from this RFC explanation. I am not sure if there is any consensus around whether `{{input}}` is good or bad, but it doesn't need to show up in this explanation.

I will also add a note to this RFC regarding the naming of Handlebars.